src/ddmd/hdrgen.d

```diff
@@ -2483,7 +2486,7 @@ public:
 
     override void visit(StructLiteralExp e)
     {
-        buf.writestring(e.sd.toChars());
+        buf.writestring(e.sd.type.toChars());
```

Comment:
That's the qualified typename or why is this needed?

Reply:
Yes that's the qualified type name. toChars is simply template instance name.